docs/pages/eas-update/debug.mdx

```diff
@@ -158,3 +158,319 @@ After publishing, the output will display the branch and the runtime version. Th
 ## Video walkthrough
 
 <ContentSpotlight url="https://www.youtube.com/embed/m9PLTr3t3S4" />
+
+## General strategies
+
+Try these strategies before using the more specific ones mentioned in this guide.
+
+### Use `expo-dev-client`
+
+Create a [development version of our build](/eas-update/expo-dev-client). It will help us preview published updates inside a problematic build.
```

Comment:
This is more for debugging the contents of the update itself as opposed to the update mechanism/configuration. In fact, we explicitly disable the update mechanism/configuration in dev clients to not give false hope that they are working just because they work in a dev client. I think this is an important distinction.